raiden/connection_manager.py

```diff
@@ -126,34 +143,48 @@ def leave(self, wait_for_settle=True, timeout=30):
                 c.partner_address) for c in open_channels]
             for channel in channel_specs:
                 try:
-                    self.api.close(*channel),
-                except RuntimeError:
+                    self.api.close(*channel)
+                except:
```

Comment:
> an explicit exception type would be less misleading

unfortunately the range of exceptions here is pretty broad (depending on the type of backend, some are from `raiden.tests` classes). I was specific on `RuntimeError` before, but that broke `tester` tests :(